backends/tofino/CMakeLists.txt

```diff
@@ -85,8 +82,7 @@ else()
     set (BFN_P4C_GIT_SHA $ENV{BFN_P4C_GIT_SHA})
   endif()
 endif()
-set (ENV{P4C_VERSION} "${BFN_P4C_VERSION} (SHA: ${BFN_P4C_GIT_SHA})")
```

Comment:
Is it possible this was used by some sort of version header generator that would now be broken?

Reply:
iirc I checked but didn't see anything. In any case P4C's version should probably override this particular version. (or P4C's version should not be overridden by this back end)